test/runtests.jl

```diff
@@ -1,7 +1,9 @@
-using ControlTimingSafety
 using Test
 using ControlSystemsBase
 
+using Revise
+using ControlTimingSafety
+
```

Comment:
There's no need to import the package under test in the unit tests, that happens automatically.  And Revise.jl isn't a dependency (nor should it be!).  Please remove both these, and then I hope the tests will pass in CI.